datadog/resource_datadog_security_monitoring_default_rule.go

```diff
@@ -112,9 +112,11 @@ func securityMonitoringRuleDeprecationWarning(rule securityMonitoringRuleRespons
 		warning := diag.Diagnostic{
 			Severity: diag.Warning,
 			Summary:  fmt.Sprintf("Rule will be deprecated on %s.", deprecation.Format("Jan _2 2006")),
-			Detail: "Please consider deleting the associated resource. " +
-				"After the deprecation date, the rule will stop triggering signals. " +
-				"Moreover, the API will reject any call to update the rule, which might break your Terraform pipeline.",
+			Detail: "Please consider deleting the associated resource." +
```

Comment:
You may want to add a space at end of each line, so there a space between period and next sentence when concatenated.